<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>949</width>
    <height>595</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>691</width>
      <height>421</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="VideoOutput">
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
    </layout>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>-2</x>
      <y>480</y>
      <width>951</width>
      <height>90</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">*{
	border:none;
}

QPushButton:hover
{
	
	background-color: rgb(255, 255, 255);
}</string>
    </property>
    <widget class="QPushButton" name="ButStop">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>20</y>
       <width>32</width>
       <height>32</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/icon/stop.png);



</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="ButPlay">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>20</y>
       <width>32</width>
       <height>32</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="ButNext">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>20</y>
       <width>32</width>
       <height>32</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/icon/next.png);


</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="ButPre">
     <property name="geometry">
      <rect>
       <x>13</x>
       <y>20</y>
       <width>32</width>
       <height>32</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/icon/pre.png);



</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="ButLoad">
     <property name="geometry">
      <rect>
       <x>600</x>
       <y>20</y>
       <width>32</width>
       <height>32</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/icon/opendir.png);

</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="ButMenu">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>20</y>
       <width>32</width>
       <height>32</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/icon/menu.png);

</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="widget_2" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>420</y>
      <width>691</width>
      <height>60</height>
     </rect>
    </property>
    <widget class="QSlider" name="SliderTime">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>631</width>
       <height>20</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </widget>
   <widget class="QListWidget" name="Playlist">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>0</y>
      <width>261</width>
      <height>481</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Ignored">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>949</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
